scripts/build_deb.sh

```diff
@@ -1,10 +1,3 @@
 #!/bin/bash
 
-rm -rf ./build/
-mkdir -p ./build/tmp/
-
-cp -r debian ./build/tmp/
-cp clickable.py ./build/tmp/clickable
-
-cd ./build/
 docker run -v `pwd`:`pwd` -w `pwd` -u `id -u` --rm -it clickable/build-deb:python3 bash -c "cd tmp && dpkg-buildpackage"
```

Comment:
By the way, I suggest using whalebuilder for building the deb, it's a wrapper for easily building debian packages in docker containers.

Reply:
Cool, I've never head of it before, I'll have to check it out!